core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1076,7 +1076,91 @@ public String toString() {
           .setScope(Scope.SERVER)
           .build();
 
-  //
+  public static final PropertyKey UNDERFS_CEPHFS_AUTH_ID =
+      new Builder(Name.UNDERFS_CEPHFS_AUTH_ID)
+          .setDefaultValue("admin")
+          .setDescription("Ceph client id for authentication.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.SERVER)
+          .build();
+  public static final PropertyKey UNDERFS_CEPHFS_CONF_FILE =
+      new Builder(Name.UNDERFS_CEPHFS_CONF_FILE)
+          .setDefaultValue("conf")
```

Comment:
> is `conf` the correct default value? you don't need to specify a default value for a property if there is not a good default value. Typically, we only set a default value here if that value will really work as default, rather than a placeholder

the default value of conf is "/etc/ceph/ceph.conf", i will modify it